_includes/v23.1/known-limitations/sql-cursors.md

```diff
@@ -0,0 +1,25 @@
+CockroachDB implements SQL {% if page.name == "known-limitations.md" %} [cursor](cursors.html) {% else %} cursor {% endif %} support with the following limitations:
+
+- `DECLARE` only supports forward cursors.
+- `FETCH` supports forward, relative, and absolute variants, but only for forward cursors.
+- `BINARY CURSOR`, which returns data in the Postgres binary format, is not supported.
+- `MOVE`, which allows advancing the cursor without returning any rows, is not supported.
+- `WITH HOLD`, which allows keeping a cursor open for longer than a transaction by writing its results into a buffer, is accepted as valid syntax within a single transaction but is not supported. It acts as a no-op and does not actually perform the function of `WITH HOLD`, which is to make the cursor live outside its parent transaction.
```

Comment:
I think important to stress here that if you are using WITH HOLD you will be forced to close that cursor within the transaction it was created in.

Reply:
Thanks, I added this info so it's super clear (hopefully)!